source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -106,6 +177,7 @@ bool AggressiveDCEPass::IsStructuredIfOrLoopHeader(ir::BasicBlock* bp,
                                                    ir::Instruction** mergeInst,
                                                    ir::Instruction** branchInst,
                                                    uint32_t* mergeBlockId) {
+  if (!bp) return false;
```

Comment:
Should update the method comment to say this method returns false if bp is null

Reply:
Ok